pkg/vm/ch.go

```diff
@@ -179,6 +186,15 @@ func (m *Machine) findVirtioFsMount() (*VirtioFS, error) {
 	return root, nil
 }
 
+func (m *Machine) copyCorex() error {
+	fs, err := m.findVirtioFsMount()
+	if err != nil {
+		return err
+	}
+	root := fs.Path
+	err = cp.Copy("/usr/bin/corex", filepath.Join(root, "/bin/corex"))
```

Comment:
Why do u need a new module to do a file copy, u can just open the 2 files, and do io.Copy().

Also make sure that `<root>/bin` directory exists.